assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -620,25 +619,8 @@ export const MESSAGES = {
         </ul>
       ),
     },
-    VIDEO_RESOLUTION_TOO_HIGH: {
-      MAIN_TEXT: sprintf(
-        /* translators: %s: maximum video resolution. */
-        __('Reduce video resolution to less than %s', 'web-stories'),
-        `${MAX_VIDEO_RESOLUTION}p`
-      ),
-      HELPER_TEXT: (
-        <ul>
-          <li>
-            {__(
-              "Optimize reach and engagement by accounting for the large number of mobile devices don't support video resolutions larger than 4K",
-              'web-stories'
-            )}
-          </li>
-        </ul>
-      ),
-    },
     VIDEO_NOT_OPTIMIZED: {
-      MAIN_TEXT: __('Video not optimized', 'web-stories'),
+      MAIN_TEXT: __('Optimize video size', 'web-stories'),
```

Comment:
i think the way you adjusted this is fine, but the text you added in the component could have also lived here as `HELPER_TEXT` to keep with the reusable pattern. Just a heads up for next time. Sorry I didn't catch it earlier!

Reply:
Good idea, done.